src/components/EpicDetailView/EpicDetailView.tsx

```diff
@@ -37,11 +46,37 @@ export function EpicDetailView({
   description: string
   created: string
   updated: string
+  status: string
+  projectId: string
+  type: string
   closeModal: () => void
 }) {
   const queryClient = useQueryClient()
   const reloadEpics = () => queryClient.invalidateQueries({ queryKey: ["epics"] });
 
+  const [defaultStatus, setDefaultStatus] = useState(status)
+  const statusMutation = useMutation({
+    mutationFn: (targetStatus: string) => setStatus(issueKey, targetStatus),
+    onSuccess: reloadEpics,
```

Comment:
Reloading the epics does not seem to work. To reproduce, open a epic, change its status and close and reopen its detail view. The status should already have changed, yet it does not.